keyboards/synthlabs/065/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 Aaron Hong (@hongaaronc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Backlighting */
+#define RGB_MATRIX_LED_COUNT 83
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
```

Comment:
```suggestion
#define RGB_MATRIX_KEYPRESSES
#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
```

Reply:
Ah yeah, I see these are redundant since already defined in `info.json` (after applying your `sleep` recommendation)... Yep make sense - applied!